Display the active language's instrument labels in the first row in the instrument page

PouyaMohseni opened the pull request:
The list of labels on the instrument page is ordered by creation time. The label for the active language is prioritized by placing it at the top of the list.

refs: #410

dchiller commented on web-app/django/VIM/apps/instruments/views/instrument_detail.py:
```diff
@@ -47,12 +47,17 @@ def get_context_data(self, **kwargs):
+        # Get the instrument label in the active language and moving it to the top of label_aliases_dict
         # Set label to the first instrument name added in the language if there is no "umil_label" set
-        active_labels = instrument_names.filter(language=context["active_language"])
+        active_lang = context["active_language"]
```
Rather than adding this to the `context` and then getting it, probably clearer to set the variable first and then add to `context`:


```suggestion
        active_lang = (
Language.objects.get(en_label=active_language_en)
if active_language_en
else Language.objects.get(en_label="English")  # default in English
)

        # Get the instrument label in the active language and moving it to the top of label_aliases_dict
# Set label to the first instrument name added in the language if there is no "umil_label" set
        context["active_language"] = active_lang
```

yinanazhou commented:
Minor improvements: 
- use `refs` in footers. 
- if you want to use 3 separate commits for this PR, the last one should be a `refactor`.

Otherwise, I believe this is good to go.

dchiller commented on web-app/django/VIM/apps/instruments/views/instrument_detail.py:
```diff
@@ -37,25 +37,31 @@ def get_context_data(self, **kwargs):
             Language.objects.get(en_label=active_language_en)
             if active_language_en
             else Language.objects.get(en_label="English")  # default in English
         )
```
```suggestion
active_language_en = self.request.session.get("active_language_en", "English")
        active_lang = Language.objects.get(en_label=active_language_en)
```

yinanazhou commented:
The PR description still uses github keyword. And the first commit is not linking the related issue correctly. Otherwise, this looks good to me.

PouyaMohseni commented:
Ok, I will highlight the out-of-order row if it exists. The ordering is not changed in this PR, but in #381. I think this would be relevant if we close #381 and I add language ordering in this PR.

PouyaMohseni committed:
fix: reorder instrument names based on creation date

PouyaMohseni committed:
fix: reorder instrument names based on creation date

PouyaMohseni committed:
fix: add a remove btn when image is added in the create page

- Add a sentence to guide in selecting files when a file is already selected.

resolves #535

PouyaMohseni committed:
fix: add a remove btn when image is added in the create page

- Add a sentence to guide in selecting files when a file is already selected.

resolves #535
